src/Framework/Telemetry/TelemetryConstants.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+namespace Microsoft.Build.Framework.Telemetry;
+
+/// <summary>
+/// Constants for VS OpenTelemetry for basic configuration and appropriate naming for VS exporting/collection.
+/// </summary>
+internal static class TelemetryConstants
+{
+    /// <summary>
+    /// "Microsoft.VisualStudio.OpenTelemetry.*" namespace is required by VS exporting/collection.
+    /// </summary>
+    public const string ActivitySourceNamespacePrefix = "Microsoft.VisualStudio.OpenTelemetry.MSBuild.";
+
+    /// <summary>
+    /// Namespace of the default ActivitySource handling e.g. End of build telemetry.
+    /// </summary>
+    public const string DefaultActivitySourceNamespace = $"{ActivitySourceNamespacePrefix}Default";
+
+    /// <summary>
+    /// Prefix required by VS exporting/collection.
+    /// </summary>
+    public const string EventPrefix = "VS/MSBuild/";
+
+    /// <summary>
+    /// Prefix required by VS exporting/collection.
+    /// </summary>
+    public const string PropertyPrefix = "VS.MSBuild.";
+
+    /// <summary>
+    /// For VS OpenTelemetry Collector to apply the correct privacy policy.
+    /// </summary>
+    public const string VSMajorVersion = "17.0";
+
+    /// <summary>
+    /// Sample rate for the default namespace.
+    /// 1:25000 gives us sample size of sufficient confidence with the assumption we collect the order of 1e7 - 1e8 events per day.
+    /// </summary>
+    // public const double DefaultSampleRate = 4e-5;
+    public const double DefaultSampleRate = 1.0;
```

Comment:
Very opinionted suggestion :-) - let's keep this as it was. During testing with A/B experiments can set env vars for both - opt-in and the override rate. Once we exit the VS experimental tasting we won't have risk of forgetting this in and storming tel backend

Reply:
The idea of the changed PR is that opt in = Override sample rate env var. So this is now a dead variable waiting to be rolled back to when "Default" means something. 
I did not want to split these concepts because override without opt in is meaningless and it's not transparent what the sample rate should be when you opt in without specifying an override (I would personally expect it to be 1, not 1:25000).

I'm not concerned it would spam the backend because for enabling the behavior by default we'd have to change MSBuild, not the VS A/B code.

Reply:
i made the change, it makes sense